target/cheri-common/cheri_tagmem.c

```diff
@@ -433,7 +564,14 @@ void cheri_tag_set(CPUArchState *env, target_ulong vaddr, int reg, hwaddr* ret_p
     qemu_maybe_log_instr_extra(env, "    Cap Tag Write [" TARGET_FMT_lx "/"
         RAM_ADDR_FMT "] %d -> 1\n", vaddr, ram_offset,
         tag_bit_get(ram_offset >> CAP_TAG_SHFT, ram));
-    tag_bit_set(ram_offset >> CAP_TAG_SHFT, ram);
+    if (tag_bit_set(ram_offset >> CAP_TAG_SHFT, ram)) {
```

Comment:
Is the comment at the start of this function still true? It could certainly do with de-MIPS-ifying.

Reply:
It does currently bypass the soft-TLB, but we should really change that. I'll update the comment once that is the case.